mgmt_cli_test.py

```diff
@@ -83,8 +83,7 @@ def restore_backup(self, mgr_cluster, snapshot_tag, keyspace_and_table_list):
         for node in self.db_cluster.nodes:
             install_dependencies(node=node, destination=self.DESTINATION)
             node_data_path = '/var/lib/scylla/data'
-            nodetool_info = self.db_cluster.get_nodetool_info(node)
-            node_id = nodetool_info['ID']
+            node_id = self.db_cluster.get_node_id(node)
```

Comment:
can you put this fix in it's own commit ?
with the proper information and a link to the the bug related to nodetool info ?

Reply:
also please update this PR title and description, maybe something like:
`refactor(manager): change TaskStatus to stop using Enum`